src/components/CoinTable/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Coin } from '../../types/types';
+import styles from './CoinTable.module.scss';
+
+const {
+  coins,
+  coins__table,
+  coins__name,
+  coins__symbol,
+  coins__price,
+  coins__volume,
+  coins__rank,
+} = styles;
+
+type CoinTableProps = {
+  coinList: Coin[];
+};
+
+class CoinTable extends React.Component<CoinTableProps> {
+  constructor(props: CoinTableProps) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <>
+        <div className={coins}>
+          <table className={coins__table}>
+            <thead>
+              <tr>
+                <th>Name</th>
+                <th>Symbol</th>
+                <th>Price</th>
+                <th>Volume (24H)</th>
+                <th>Rank</th>
+              </tr>
+            </thead>
```

Comment:
Why did you opt for the older `<table>` markup instead of more modern `css/grid`? Grid behaves better when localization is concerned (e.g. in `ltr` languages) and is more easily configurable